tests/serve/test_sglang.py

```diff
@@ -116,6 +118,39 @@ class SGLangConfig(EngineConfig):
             )
         ],
     ),
+    "embedding_agg": SGLangConfig(
+        name="embedding_agg",
+        directory=sglang_dir,
+        script_name="agg_embed.sh",
+        marks=[pytest.mark.gpu_1],
+        model="Qwen/Qwen3-Embedding-4B",
```

Comment:
Do we need to update the test logic to pre-download this embedding model?

ex:
- https://github.com/ai-dynamo/dynamo/blob/4d48fe67d8fe13b107237856cb652e10a8f137f0/tests/utils/constants.py#L15
- https://github.com/ai-dynamo/dynamo/blob/4d48fe67d8fe13b107237856cb652e10a8f137f0/tests/conftest.py#L109